lib/vet360/models/validation_address.rb

```diff
@@ -30,27 +30,40 @@ def address_validation_req
         )
       end
 
+      # rubocop:disable Metrics/MethodLength
       # @return [Vet360::Models::ValidationAddress] validation address model created from
       #   address validation API response
       def self.build_from_address_suggestion(address_suggestion_hash)
         address_hash = address_suggestion_hash['address']
-
-        # add international_postal_code and province for future international support
-        new(
+        attributes = {
           address_line1: address_hash['address_line1'],
           address_line2: address_hash['address_line2'],
           address_line3: address_hash['address_line3'],
           address_type: address_suggestion_hash['address_meta_data']['address_type'].upcase,
           city: address_hash['city'],
           country_name: address_hash['country']['name'],
-          country_code_iso3: address_hash['country']['iso3_code'],
-          county_code: address_hash.dig('county', 'county_fips_code'),
-          county_name: address_hash.dig('county', 'name'),
-          state_code: address_hash['state_province']['code'],
-          zip_code: address_hash['zip_code5'],
-          zip_code_suffix: address_hash['zip_code4']
+          country_code_iso3: address_hash['country']['iso3_code']
+        }
+
+        attributes.merge!(
```

Comment:
Instead of skipping the rubcop, I think collapsing the regional attributes into another method will bring the method size down and separate out branching logic.
I pulled down your branch and this is what I came up with:
```
def self.build_from_address_suggestion(address_suggestion_hash)
        address_hash = address_suggestion_hash['address']
        address_type = address_suggestion_hash['address_meta_data']['address_type'].upcase
        attributes = {
          address_line1: address_hash['address_line1'],
          address_line2: address_hash['address_line2'],
          address_line3: address_hash['address_line3'],
          address_type: address_type,
          city: address_hash['city'],
          country_name: address_hash['country']['name'],
          country_code_iso3: address_hash['country']['iso3_code']
        }.merge(regional_attributes(address_type, address_hash))

        new(attributes)
      end

      def self.regional_attributes(address_type, address_hash)
        if address_type == INTERNATIONAL
          {
            province: address_hash['state_province']['name'],
            international_postal_code: address_hash['international_postal_code']
          }
        else
          {
            state_code: address_hash['state_province']['code'],
            county_code: address_hash.dig('county', 'county_fips_code'),
            county_name: address_hash.dig('county', 'name'),
            zip_code: address_hash['zip_code5'], zip_code_suffix: address_hash['zip_code4']
          }
        end
      end
```

Reply:
thanks for the suggestions, i made the changes